scripts/build-glean-metadata.py

```diff
@@ -258,6 +258,87 @@ def get_app_variant_description(app):
             json.dumps(app.app)
         )
 
+        pings_with_client_id = []
```

Comment:
It might make slightly more sense to have a dictionary of ping names -> ping data, which we could use for both identifying client id information as well as whether a ping has been seen yet. But maybe not worth worrying about too much for now. However, at the very least we *should* use a set here (like we already do for `ping_identifiers_seen`).

```suggestion
        pings_with_client_id = set()
```